packages/runtime/runtime-utils/src/summaryUtils.ts

```diff
@@ -146,27 +146,45 @@ export function addSummarizeResultToSummary(
 	summary.stats = mergeStats(summary.stats, summarizeResult.stats);
 }
 
+/**
+ * An object who's properties are used to initialize a {@link SummaryTreeBuilder}
+ * @legacy
+ * @alpha
+ */
+export interface SummaryTreeBuilderParams {
+	/**
+	 * This value will become the {@link @fluidframework/driver-definitions#ISummaryTree.groupId}
```

Comment:
Nit: An alternative documentation strategy here would be to define the docs on `SummaryTreeBuilder.groupId`, and document this as `{@inheritDoc SummaryTreeBuilder.groupId}`.

Then both would be documented and kept in sync.

Reply:
Also, what does it mean for this to be `undefined`?